build.sbt

```diff
@@ -75,7 +75,8 @@ lazy val proto2Test: Project = Project(
   "proto2test",
   file("proto2test")
 ).enablePlugins(ProtobufPlugin).settings(
-  commonSettings ++ protoSettings ++ noPublishSettings
+  commonSettings ++ protoSettings ++ noPublishSettings,
+  protobufProtocOptions in ProtobufConfig ++= Seq("--include_std_types"),
```

Comment:
this configuration is required to add `"google/protobuf/descriptor.proto"` to the class path when running `protoc` command.